homeassistant/components/sensibo/manifest.json

```diff
@@ -2,7 +2,7 @@
   "domain": "sensibo",
   "name": "Sensibo",
   "documentation": "https://www.home-assistant.io/integrations/sensibo",
-  "requirements": ["pysensibo==1.0.3"],
+  "requirements": ["pysensibov2==1.0.1"],
```

Comment:
Did you reach out to Andrey and ask if he could transfer the project to you or if you could take over maintenance?

Reply:
Doesn't seem to be a way of contacting him (in that case I would have) and he has no activity on github for almost 4 years so I took the approach here to just move on.
He's a maintainer here also but seems not to be contributing anymore?

Reply:
Is there an email to find in an old commit?

Reply:
I went back to 2013 and found some gmail address so I wrote him an e-mail now so let's see if there is a response on it.
We can park this a few days to see if there is a response.

Reply:
I have added Goran to pysensibo on both github and pypi

Reply:
I will close the PR